includes/admin/class-wcmypa-admin.php

```diff
@@ -989,14 +990,22 @@ private function getConfirmationData(DeliveryOptions $deliveryOptions): ?array
             ];
         }
 
-        return [
-            __("Delivery type:", "woocommerce-myparcel") => WCMP_Data::getDeliveryTypesHuman()[$deliveryOptions->getDeliveryType()],
-            __("Date:", 'woocommerce')                   => wc_format_datetime(new WC_DateTime($deliveryOptions->getDate())),
-            __("Extra options:", "woocommerce-myparcel") =>
-                sprintf("%s<br>%s",
-                        $deliveryOptions->getShipmentOptions()->hasSignature() ? $signatureTitle : null,
-                        $deliveryOptions->getShipmentOptions()->hasOnlyRecipient() ? $onlyRecipientTitle : null)
+        $confirmationData = [
+            __("delivery_type:", "woocommerce-myparcel") => WCMP_Data::getDeliveryTypesHuman()[$deliveryOptions->getDeliveryType()],
```

Comment:
Ik zou misschien de `:` weg laten, achter `delivery_type`. Het probleem nu is namelijk als iemand vind dat hij weg moet dan moet er alsnog een aanpassing gedaan worden in de code.

Reply:
done